src/platform/windows/window.rs

```diff
@@ -282,7 +282,7 @@ impl Drop for Window {
     #[inline]
     fn drop(&mut self) {
         unsafe {
-            user32::PostMessageW(self.window.0, winapi::WM_DESTROY, 0, 0);
+            user32::PostMessageW(self.window.0, winapi::WM_CLOSE, 0, 0);
```

Comment:
Please add a comment saying that our callback will respond to `WM_CLOSE` by sending a `WM_DESTROY`.